### PR TITLE
BD-395 - Currents Revision

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm...

1. Moving Currents to the User Guide
2. Adding metadata and applying templates to the pages
3. Revising the Events listings as searchable glossaries
4. Setting up redirects to the proper locations in Currents

### CODE REVIEW

File: _docs/_help/help_articles/data/dispatch_id.md [hunk 1]
```diff
@@ -43,4 +43,4 @@ The `dispatch_id` is the unique id generated for each transmission – or, dispa
...
+You can use `dispatch_id` as a [Personalization Tag]({{ site.baseurl }}/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/#supported-personalization-tags), in [Message Engagement Events]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/data_storage_events/message_engagement_events/), or when you use [Segment]({{ site.baseurl }}/partners/technology_partners/data_and_infrastructure_agility/customer_data_platform/segment_for_currents/#integration-details), [Mixpanel]({{ site.baseurl }}/partners/technology_partners/insights/behavioral_analytics/mixpanel_for_currents/#email-events), or [Amplitude]({{ site.baseurl }}/partners/technology_partners/insights/behavioral_analytics/amplitude_for_currents/#email-events) for Currents.
```
Comment (KellieHawks) [inline _docs/_help/help_articles/data/dispatch_id.md]: message engagement link directing to 404 page

File: _docs/_help/release_notes/2019/april.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ page_order: 9
...
+In addition to some corrections to the section, a new [Subscription Event]({{ site.baseurl}}/user_guide/data_and_analytics/braze_currents/data_storage_events/message_engagement_events/#subscription-events) has been added to the Message Engagement Events page. You can now export Subscription Group State Change data from Braze to [Segment]({{ site.baseurl}}/partners/technology_partners/data_and_infrastructure_agility/customer_data_platform/segment_for_currents/#integration-details) and [mParticle]({{ site.baseurl}}/partners/technology_partners/data_and_infrastructure_agility/customer_data_platform/mparticle_for_currents/#integration-details), as well as that and Install Attribution Events in [Mixpanel]({{ site.baseurl}}/partners/technology_partners/insights/behavioral_analytics/mixpanel_for_currents).
```
Comment (KellieHawks) [inline _docs/_help/release_notes/2019/april.md]: subscription event link directs to 404 link

File: _docs/_help/release_notes/2019/april.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ page_order: 9
...
 
-Additionally, the property `canvas_step_id` has been added to available [Conversion Events]({{ site.baseurl}}/partners/braze_currents/data_storage_events/message_engagement_events/#conversion-events).
+Additionally, the property `canvas_step_id` has been added to available [Conversion Events]({{ site.baseurl}}/user_guide/data_and_analytics/braze_currents/data_storage_events/message_engagement_events/#conversion-events).
```
Comment (KellieHawks) [inline _docs/_help/release_notes/2019/april.md]: conversions events link directs to 404 link

File: _docs/_help/release_notes/2019/october.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Canvas has new and improved analytics to view the performance of each of your va
 
 ## SMS
 
-Braze now provides SMS with Campaigns, Canvas, and [Currents]({{ site.baseurl }}/partners/braze_currents/data_storage_events/message_engagement_events/)! Check out our [Set Up Guide]({{ site.baseurl }}/user_guide/onboarding_with_braze/sms_setup/) to get started and our [SMS Sending Guide]({{ site.baseurl }}/user_guide/message_building_by_channel/sms/) to learn more!
+Braze now provides SMS with Campaigns, Canvas, and [Currents]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/data_storage_events/message_engagement_events/)! Check out our [Set Up Guide]({{ site.baseurl }}/user_guide/onboarding_with_braze/sms_setup/) to get started and our [SMS Sending Guide]({{ site.baseurl }}/user_guide/message_building_by_channel/sms/) to learn more!
```
Comment (KellieHawks) [inline _docs/_help/release_notes/2019/october.md]: currents link directs to 404

File: _docs/_user_guide/data_and_analytics/braze_currents/setting_up_currents.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+![Adding an Integration]({% image_buster /assets/img/new_current.png %}){: height="50%" width="50%"}
+
+Each partner requires a different set of configuration steps. To enable each integration, [see the instructions in their respective pages]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/available_partners/).
+
+## Step 4: Configure Events
+
+Choose the events you wish to pass to that partner by checking from the available options. You can find listings of these events in our [Customer Behavior Events]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/customer_behavior_events/) and [Message Engagement Events]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/message_engagement_events/) libraries.
+
+If needed, you can learn more about our events in our [Event Delivery Semantics]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/event_delivery_semantics/) article.
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/braze_currents/setting_up_currents.md]: event delivery semantics link directs to 404

File: _docs/_user_guide/data_and_analytics/braze_currents/s3_to_snowflake.md [hunk 6]
```diff
@@ -105,20 +110,20 @@ In AWS, navigate to the corresponding bucket of the new Snowflake stage. Then, u
 
 ![AWS Properties][1]{: height="50%" width="50%"}
 
-In **Events**, create new events for each set of Currents Data, as needed ([Messaging]({{ site.baseurl }}/partners/braze_currents/data_storage_events/message_engagement_events/) and/or [User Behavior]({{ site.baseurl }}/partners/braze_currents/data_storage_events/customer_behavior_events/)).
+In **Events**, create new events for each set of Currents Data, as needed ([Messaging]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/data_storage_events/message_engagement_events/) and/or [User Behavior]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/data_storage_events/customer_behavior_events/)).
 
 ![AWS Events][2]{: height="50%" width="50%"}
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/braze_currents/s3_to_snowflake.md]: could not directly see the link that is broken but...
(messaging and user behavior) links both direct to 404 link

File: _docs/_user_guide/data_and_analytics/braze_currents/s3_to_snowflake.md [hunk 7]
```diff
@@ -105,20 +110,20 @@ In AWS, navigate to the corresponding bucket of the new Snowflake stage. Then, u
...
 
 It is critical that your tables are structured in accordance to the Braze Currents schemas, as Braze Currents will continuously load data into them via specific fields with specific data types (a `user_id` will always be loaded as a string and called a `user_id` in Currents data).
 
 {% alert note %}
-  Depending on your Currents integration, you may have different events you must set up ([Message Engagement or Messaging Events]({{ site.baseurl }}/partners/braze_currents/data_storage_events/message_engagement_events/), [User or Customer Behavior Events]({{ site.baseurl }}/partners/braze_currents/data_storage_events/customer_behavior_events/), or both).  You can also write a script for some or all of this process.
+  Depending on your Currents integration, you may have different events you must set up ([Message Engagement or Messaging Events]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/data_storage_events/message_engagement_events/), [User or Customer Behavior Events]({{ site.baseurl }}/user_guide/data_and_analytics/braze_currents/data_storage_events/customer_behavior_events/), or both).  You can also write a script for some or all of this process.
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/braze_currents/s3_to_snowflake.md]: both of these links direct to 404 page.
message engagement or messaging events
user or customer behavior events

**Comment by dulle-braze** [comment]
Still getting some 404s:

- https://www.braze.com/docs/partners/technology_partners/data_and_infrastructure_agility/data_warehouses/snowflake/, which is the first google result when you search Braze Data Sharing
- https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents/event_delivery_semantics/ (clicking the Event Delivery Semantics "button" on the main Currents page)

**Pull request merged**